tests/tasks-sqs/src/main/java/example/TasksPublisher.java

```diff
@@ -23,7 +23,7 @@ public void sendMessages() {
         try {
             int id = tasksIds.incrementAndGet();
             LOGGER.info("Publishing a task with id: " + id);
-            tasksProducer.send(new Task(id), "" + id);
+            tasksProducer.send(new Task(id));
```

Comment:
Why was this change necessary?

Reply:
SQS messages were failing complaining about the message header id but the error message wasn't helpful. turns out it was because that header is only valid with FIFO queues but the queue was a regular queue with a FIFO name. I updated queue creation and restored the header